Source/Core/DolphinQt2/Config/FilesystemWidget.h

```diff
@@ -7,7 +7,10 @@
 #include <QDialog>
 #include <memory>
 
+#ifndef Q_MOC_RUN
```

Comment:
This should probably have a comment around it explaining why it in particular is left out, so its easier for others to assess if the ifdef to be removed some time in the future.